pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -73,6 +75,7 @@ type EditOptions struct {
 	ErrOut io.Writer
 
 	f                   cmdutil.Factory
+	file                string
```

Comment:
moving this here and mutating it within the edit loop makes it very hard to reason about the edit options... they should not mutate as part of running the command